docs/release-notes/5.7.43-31.65.md

```diff
@@ -0,0 +1,29 @@
+# Percona XtraDB Cluster 5.7.43-31.65 (2023-08-)
+
+Percona XtraDB Cluster (PXC) supports critical business applications in your public, private, or hybrid cloud environment. Our free, open source, enterprise-grade solution includes the high availability and the security features your business requires to meet your customer expectations and business goals.
+
+## Release highlights
+
+Improvements and bug fixes provided by Oracle for MySQL 5.7.43 and included in Percona XtraDB Cluster are the following:
+
+* OpenSSL 1.1.1 library has been upgraded to OpenSSL 3.0.9.
+
+Find the full list of bug fixes and changes in the [MySQL 5.7.43 Release Notes](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-43.html).
+
+## Bug fixes
+
+* [PXC-4218](https://jira.percona.com/browse/PXC-4218): The output of SHOW VARIABLES LIKE 'wsrep_provider_options' was truncated.
```

Comment:
I recommend saying explicitly that due to this bug fix, it is absolutely necessary to follow this upgrade procedure
https://www.percona.com/doc/percona-xtradb-cluster/5.7/howtos/upgrade_guide.html
especially point 7 is required:
`Open another session and run mysql_upgrade.`

Reply:
updated